libbeat/template/config.go

```diff
@@ -0,0 +1,17 @@
+package template
+
+type TemplateConfig struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
type name will be used as template.TemplateConfig by other packages, and that stutters; consider calling this Config